client/elements/text/sc-simple-text.html

```diff
@@ -297,17 +333,30 @@ <h3>Could not find the sutta text.</h3>
               if (textElement) {
                   visible ? textElement.classList.add('infomode') : textElement.classList.remove('infomode');
                   for (var key in this.classTitles) {
-                      const classrefs = this.$.simple_text_content.querySelectorAll('.' + key);
+                      const classrefs = textElement.querySelectorAll(`.${key}`);
                       Array.from(classrefs).forEach(item => item.title = this.classTitles[key]);
                   }
                   for (var key in this.textualInfoClassTitles){
-                      const textualInfoclassrefs = this.$.simple_text_content.querySelectorAll(`.${key}`);
+                      const textualInfoclassrefs = textElement.querySelectorAll(`.${key}`);
                       if (visible) {
                           Array.from(textualInfoclassrefs).forEach(item => item.title = this.textualInfoClassTitles[key]);
                       } else {
                           Array.from(textualInfoclassrefs).forEach(item => item.title = "");
                       }
                   }
+                  const varClasses = textElement.querySelectorAll('.var');
+                  if (varClasses[0].innerHTML.indexOf('class="varnote"') == -1) {
```

Comment:
please use the `===` operator instead of `==`.
https://stackoverflow.com/questions/523643/difference-between-and-in-javascript?noredirect=1&lq=1